proposals/p1983.md

```diff
@@ -0,0 +1,183 @@
+# Weaken digit separator placement rules
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/1983)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Problem](#problem)
+-   [Background](#background)
+-   [Proposal](#proposal)
+-   [Rationale](#rationale)
+-   [Alternatives considered](#alternatives-considered)
+    -   [3-digit decimal groupings](#3-digit-decimal-groupings)
+    -   [2-digit or 4-digit hexadecimal digit groupings](#2-digit-or-4-digit-hexadecimal-digit-groupings)
+    -   [Disallow digit separators in fractions](#disallow-digit-separators-in-fractions)
+
+<!-- tocstop -->
+
+## Abstract
+
+[Proposal #143: Numeric literals](/proposals/p0143.md) added digit separators
+with strict rules for placement. It missed some use-cases. In order to address
+this, remove placement rules for numeric literals.
+
+## Problem
+
+Digit separator placement rules may be too strict.
```

Comment:
```suggestion
Digit separator placement rules are too strict.
```

I would also suggest moving the issues raised out of background and put them here.